### DISCUSSION TITLE
加个翻译失败自动重试,,总弹弹窗像高速下崽器

### DISCUSSION THREAD

**Discussion by cancundeyingzi** [immersive-translate/immersive-translate#1304] [Ideas]
### 新功能 | New Feature

1.翻译失败自动重试
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.总弹弹窗像高速下崽器
2.
3.
![image](https://github.com/immersive-translate/immersive-translate/assets/73635883/3cee0efc-b61a-4e78-a786-897237db339a)

### 提议 | Proposal
或者小的重试按钮就可以了,不要那么大一个......
_No response_

**Comment by theowenyoung** [comment]
你会经常遇到翻译错误吗？

**Reply by cancundeyingzi** [reply]
部分时间段会总是遇到.应该受到了网络波动的影响

**Reply by theowenyoung** [reply]
这个是什么翻译服务？错误有点奇怪。

**Reply by theowenyoung** [reply]
是整页遇到错误吗？

目前的设定是 超过 10个错误才会主动弹窗

**Reply by cancundeyingzi** [reply]
![image](https://github.com/immersive-translate/immersive-translate/assets/73635883/c27d363e-e656-446f-9e89-34f2ff0173be)

**Reply by cancundeyingzi** [reply]
使用alt+a整页翻译
